File: rng.py
import random

def random_number_generator(min_value, max_value, seed, count):
    random.seed(seed)
    return [random.randint(min_value, max_value) for _ in range(count)]

# 使用範例
min_value = 1
max_value = 100
seed = 42
count = 5

random_numbers = random_number_generator(min_value, max_value, seed, count)
print(random_numbers)

File: rng.js
function randomInteger(min, max) {
    return Math.floor(Math.random() * (max - min + 1)) + min;
}

function randomNumberGenerator(minValue, maxValue, count) {
    let numbers = [];
    for (let i = 0; i < count; i++) {
        numbers.push(randomInteger(minValue, maxValue));
    }
    return numbers;
}

// 使用範例
const minValue = 1;
const maxValue = 100;
const count = 5;

const randomNumbers = randomNumberGenerator(minValue, maxValue, count);
console.log(randomNumbers);
